packages/dumb/src/provider/trends.provider.hook.tsx

```diff
@@ -69,13 +87,44 @@ export const useTrendsProviderHook = (): TrendsContextResult => {
     }
   }
 
-  const focusCbgDateTrace = (cbgDateTrace: CbgDateTrace, position: CbgPositionData): void => {
+  const focusCbgDateTrace = (cbgDateTrace: CbgDateTrace, position: CbgPositionData<CbgDateTraceYPositions>): void => {
     setFocusedCbgDateTrace({ data: cbgDateTrace, position })
   }
 
   const unfocusCbgDateTrace = (): void => {
     setFocusedCbgDateTrace(undefined)
   }
 
-  return { displayFlags, focusCbgDateTrace, focusedCbgDateTrace, toggleCbgSegments, unfocusCbgDateTrace }
+  const focusCbgSlice = (cbgSlice: CbgSlice, position: CbgPositionData<CbgSliceYPositions>, keys: RangeSegmentSlice[]): void => {
+    const newFocusedCbgSlice = { data: cbgSlice, position, keys }
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏
```suggestion
    const newlyFocusedCbgSlice = { data: cbgSlice, position, keys }
```